samza-core/src/main/java/org/apache/samza/clustermanager/AbstractContainerAllocator.java

```diff
@@ -97,7 +94,7 @@ public AbstractContainerAllocator(ClusterResourceManager containerProcessManager
   }
 
   /**
-   * Continually schedule StreamProcessors to run on resources obtained from the cluster manager.
+   * Continually schedule processors to run on resources obtained from the cluster manager.
```

Comment:
* Continually schedule processors to run on containers obtained from the cluster manager.